docs/playground.md

```diff
@@ -0,0 +1,267 @@
+---
+slug: playground
+append_help_link: true
+title: Semgrep Playground
+description: "Semgrep Playground is a beginner-friendly environment for creating your first rules."
+hide_title: true
+
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Creating rules with Semgrep Playground
+
+
+Semgrep Playground is a live editor used to create and test rule patterns on sample code. By testing the pattern out on sample code, you are able to quickly assess the purpose, utility, and speed of a rule as well as save it for later refinement, reuse, or for sharing.
+
+
+## Creating a rule
+
+There are two ways to create a rule:
+
+<dl>
+    <dt>Simple view</dt>
+    <dd>The simple view is best for quick and simple pattern-matching use cases, but does not display all Semgrep operators.</dd>
+    <dt>Advanced view</dt>
+    <dd>The advanced view provides the minimum required keys for a Semgrep rule and expects users to fill in additional keys, such as pattern operators or metadata, to complete the rule.</dd>
```

Comment:
```suggestion
    <dd>The advanced view provides the minimum required keys for a Semgrep rule. To complete the rule, the advanced view requires users to fill in additional keys such as pattern operators or metadata.</dd>
```

This might be slightly easier to read. Do we want to use word key and keys or a parameter and parameters? Not sure if we have to agree upon one now, but might save us additional work. If the word key is used in the app, use key of course. The key has many meanings, see: https://developers.google.com/style/word-list?hl=en (search for key)
![Screenshot 2022-02-07 at 13 30 24](https://user-images.githubusercontent.com/26669307/152788436-b074326c-bb96-4f17-ae32-a261ee15876f.jpg)